pkgs/development/python-modules/fvcore/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, iopath
+, numpy
+, pillow
+, pyyaml
+, shapely
+, tabulate
+, termcolor
+, torch
+, torchvision
+, tqdm
+, yacs
+}:
+
+buildPythonPackage rec {
+  pname = "fvcore";
+  disabled = pythonOlder "3.6";
+  version = "0.1.5";
+
+  src = fetchFromGitHub {
+    owner = "facebookresearch";
+    repo = "fvcore";
+    rev = "6bde555";
+    sha256 = "1vz9qif5iaxhwks5mhg9lrfr16aj5kb108dgphhcv0iq4nla2wsa";
+  };
+
+  nativeBuildInputs = [
+    iopath
+    numpy
+    pillow
+    pyyaml
+    tabulate
+    termcolor
+    tqdm
+    yacs
+  ];
+
+  propagatedBuildInputs = [
+    torch
+    shapely
```

Comment:
```suggestion
```

shapely seems to be an optional dependency.

Please add it to `passthrou` ([how](https://nixos.org/manual/nixpkgs/stable/#python-optional-dependencies)), then maybe ++ it to the checkinputs if needed.